Topic7/src/main/java/com/aguirre/topic7/models/StudentCourseDao.java

```diff
@@ -0,0 +1,16 @@
+package com.aguirre.topic7.models;
+
+import java.util.List;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+public interface StudentCourseDao extends JpaRepository<StudentCourse, Long> {
+
+	//@Query("SELECT registration_number FROM Student_Course WHERE id_course = :id_course")

```

Comment:
Please read this page (http://blog.netgloo.com/2014/10/27/using-mysql-in-spring-boot-via-spring-data-jpa-and-hibernate/) under the title "Data Access Object for entity User" to see how the queries are being generated